src/components/LabelTextLine.tsx

```diff
@@ -29,14 +30,14 @@ const LabelTextLine: React.FC<LabelTextLineProps> = ({ label, text, dataTestId,
   <StyledLine>
     <StyledLabel>{label}:</StyledLabel>
     {externalLink && (
-      <a href={externalLink} target="_blank" rel="noopener noreferrer">
+      <MuiLink href={externalLink} target="_blank" rel="noopener noreferrer">
```

Comment:
Kan ikke bruke `<MuiLink component={Link} to={internalLink}>` fordi den peker ut av appen?

Reply:
Stemmer